vector/src/main/res/values-fr/strings.xml

```diff
@@ -231,8 +231,8 @@ Cliquez sur exporter pour les sauvegarder avant de vous déconnecter.</string>
     <string name="room_participants_action_devices_list">Afficher la liste des appareils</string>
     <string name="people_search_local_contacts">CONTACTS LOCAUX (%d)</string>
     <string name="people_search_filter_text">Utilisateurs Matrix uniquement</string>
-    <string name="people_search_invite_by_id_dialog_title">Inviter un utilisateur par identifiant</string>
-    <string name="people_search_invite_by_id_dialog_hint">E-mail ou ID Matrix</string>
+    <string name="people_search_invite_by_id_dialog_title">Inviter un utilisateur par email</string>
```

Comment:
Maybe all the string ids should be renamed for clarity
Ex: people_search_invite_by_email_dialog_title